Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -0,0 +1,134 @@
+%{
+  from gyb_syntax_support import SYNTAX_NODES
+  from gyb_helpers import SyntaxBuildableNode
+  # -*- mode: Swift -*-
+  # Ignore the following admonition it applies to the resulting .swift file only
+}%
+//// Automatically Generated From BuildableNodes.swift.gyb.
+//// Do Not Edit Directly!
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+% for node in [SyntaxBuildableNode(node) for node in SYNTAX_NODES if node.is_buildable()]:
```

Comment:
It’s pretty much equivalent to the following Swift code
```swift
for node in SYNTAX_NODES.filter({ $0.is_buildable() }).map({ SyntaxBuildableNode($0) }) 
```

Python can essentially do filter and map combined using this list constructor.

I decided to put the `if node_is_buildable()` inside the collection that is being iterated using `for` because:
- We can’t do `if not node.is_buildable(): continue` in gyb (it’s complaining about `continue being executed while not in a loop)
- Putting the entire loop body in `if node.is_buildable()` (a) increases the indentation by one and (b) always made me wonder if there was an `else` branch to the `if`-statement further below.

Reply:
Idea/suggestion. 

Maybe it's just me that is not used to python syntax. 
But could we change `% for node in [SyntaxBuildableNode(node) for node in SYNTAX_NODES if node.is_buildable()]:` -> `% for node in [SyntaxBuildableNode(node) for syntax_node in SYNTAX_NODES if syntax_node.is_buildable()]:`

I think that two `node`s read a little hard. 
But again, maybe it's just me that's not used to this syntax :)

Reply:
Good idea. It was clear to me while I wrote it but I can see that it’s confusing when you’re reading it for the first time. Changed it.